build.psm1

```diff
@@ -1513,10 +1512,8 @@ function Install-Dotnet {
         }
 
         if ($uninstallScript) {
-            Start-NativeExecution {
-                curl -sO $obtainUrl/uninstall/$uninstallScript
-                Invoke-Expression "$sudo bash ./$uninstallScript"
-            }
+            Start-NativeExecution ([scriptblock]::Create("curl -sO $obtainUrl/uninstall/$uninstallScript"))
+            Start-NativeExecution ([scriptblock]::Create("$sudo bash ./$uninstallScript"))
```

Comment:
To be 100% clear, this is what invoke-expression does... so we are not fixing any issue here.